docker-compose.override.yaml

```diff
@@ -14,15 +14,15 @@ services:
       - ./app:/home/node/app
       - ./package.json:/home/node/package.json
     environment:
-      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-statement-generator-azurite:10000/devstoreaccount1;
+      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-statement-azurite:10000/devstoreaccount1;
 
   ffc-pay-statement-generator-postgres:
     ports:
       - "5451:5432"
     volumes:
       - postgres_data:/var/lib/postgresql/data
 
-  ffc-pay-statement-generator-azurite:
+  ffc-pay-statement-azurite:
```

Comment:
Change container to be consistent across both `publisher` and `generator` projects.  This means that both can target the same container endpoint.